code/modules/multiz/hoist.dm

```diff
@@ -0,0 +1,233 @@
+///////////////////////////
+// Dost thou even hoist? //
+///////////////////////////
+
+/obj/item/hoist_kit
+	name = "hoist kit"
+	desc = "A setup kit for a hoist that can be used to lift things. The hoist will deploy in the direction you're facing."
+	icon = 'icons/obj/hoists.dmi'
+	icon_state = "hoist_case"
+
+/obj/item/hoist_kit/attack_self(mob/user)
+	new /obj/structure/hoist (get_turf(user), user.dir)
+	user.visible_message(span("warning", "[user] deploys the hoist kit!"), span("notice", "You deploy the hoist kit!"), span("notice", "You hear the sound of parts snapping into place."))
+	qdel(src)
+
+/obj/effect/hoist_hook
+	name = "hoist clamp"
+	desc = "A clamp used to lift people or things."
+	icon = 'icons/obj/hoists.dmi'
+	icon_state = "hoist_hook"
+	var/obj/structure/hoist/source_hoist
+	can_buckle = 1
+	anchored = 1
+
+/obj/effect/hoist_hook/MouseDrop_T(atom/movable/AM,mob/user)
```

Comment:
`user` is not checked as being able or human for that matter. Use `can_use()`, easiest way.